tests/TestHelpers/GraphHelper.php

```diff
@@ -446,6 +446,60 @@ public static function getGroups(
 		);
 	}
 
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $adminUser
+	 * @param string $adminPassword
+	 * @param string $groupName
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function getGroup(
+		string $baseUrl,
+		string $xRequestId,
+		string $adminUser,
+		string $adminPassword,
+		string $groupName
+	): ResponseInterface {
+		$url = self::getFullUrl($baseUrl, 'groups/' . $groupName);
+		return HttpRequestHelper::get(
+			$url,
+			$xRequestId,
+			$adminUser,
+			$adminPassword,
+			self::getRequestHeaders()
+		);
+	}
+
+	/**
+	 * @param string $baseUrl
+	 * @param string $groupId
+	 * @param string $xRequestId
+	 * @param string $user
+	 * @param string $password
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function getSingleGroup(
+		string $baseUrl,
+		string $groupId,
+		string $xRequestId,
+		string $user,
+		string $password
+	): ResponseInterface {
+		$url = self::getFullUrl($baseUrl, 'groups/' . $groupId);
+		return HttpRequestHelper::get(
+			$url,
+			$xRequestId,
+			$user,
+			$password,
+			self::getRequestHeaders(),
+		);
+	}
```

Comment:
duplicate method, `getGroup` can be used
```suggestion
```

Reply:
I had my doubts here. two same methods: `graph/v1.0/groups/id` and `graph/v1.0/groups/groupName`
maybe keep both? because web use `graph/v1.0/groups/id` and `graph/v1.0/groups/groupName` method helper which get us id

Reply:
Ohh, didn't notice the difference :smile: 
Thanks